src/test/resources/com.epam.tc.hw5/cucumber/UserTablePage.feature

```diff
@@ -0,0 +1,25 @@
+Feature: User Table page
+
+  Scenario: User Table Page test
+    Given I open JDI GitHub site
+    And I login as user 'Roman Iovlev'
+    When I click on 'Service' button in Header
```

Comment:
The main purpose is to get all test data from the steps in every test. Need to refactor code depends on new approach.